recipe/meta.yaml

```diff
@@ -18,6 +18,10 @@ requirements:
     - pip
   run:
     - python >=3.8,<3.12
+    - notebook
+    - r-irkernel
```

Comment:
@ocefpaf What was the motivation for adding `r-irkernel` as a dependency? This is pulling in quite the dependency tree now for me, including a full R install. As far as I can tell, this is not needed for `nb_conda_kernels` to import and work properly in general, and isn't included in upstream's in-repo conda recipe.

Reply:
Agreed, this should not be a dependency.